### PR TITLE
Adding Nancy

### PR DESCRIPTION
**Pull request by ngranda**

### CODE REVIEW

File: source/people/main.rst [hunk 1]
```diff
@@ -154,4 +155,4 @@ Alumni
    kiesling
    jacobson
    opotowsky
-   ../jobs/programmer-2012
+   ../jobs/programmer-2010
```
Comment (gonuke) [inline source/people/main.rst]: Is this change intentional?  Or does something need a rebase?

**Comment by ngranda** [comment]
Not intentional. Sorry about that.

File: source/people/main.rst [hunk 2]
```diff
@@ -155,3 +157,4 @@ Alumni
    jacobson
    opotowsky
    ../jobs/programmer-2012
+   ../jobs/programmer-2010
```
Comment (gonuke) [inline source/people/main.rst]: This was added accidentally as well... as part of the same change you already fixed

**Comment by gonuke** [comment]
And you're now officially a CNERG-member
